.github/workflows/aws-deploy.yml

```diff
@@ -90,6 +90,37 @@ jobs:
           cdk_subcommand: 'deploy'
           cdk_args: '--require-approval never --context env=staging --progress events --debug'
           actions_comment: false
+
+  cdk-deploy-dev-refactor:
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/dev' }}
```

Comment:
shouldn't this execute based on a different branch?

Reply:
Yes. Thank you for the comment @zaro0508 !